lomap/dbmol.py

```diff
@@ -68,10 +68,11 @@ class DBMolecules(object):
     """
 
     # Initialization function
+    #sliu 07/17 add the radial graph option
```

Comment:
Don't store history about what you changed and when into the code itself. GitHub keeps track of this for you. No need for this type of comment.